cardea/problem_definition/readmission.py

```diff
@@ -0,0 +1,144 @@
+import pandas as pd
+
+from cardea.problem_definition import ProblemDefinition
+
+
+class Readmission (ProblemDefinition):
+    """Defines the problem of length of stay, finding how many days
+        the patient will be in the hospital.
+
+        Attributes:
+        target_label: The target label of the prediction problem.
+        target_entity: The entity name in which it contains the target label.
+        cutoff_time_label: The cutoff time label of the prediction problem
+        cutoff_entity: The entity name in which it contains the cutoff time label.
+        prediction_type: The type of the machine learning prediction.
```

Comment:
@ShahadThobaiti please include the assumptions you mentioned to me in this docstring.
- the patient visit is considered a readmission if he visits the hospital again within 30 days.
- the readmission diagnosis does not have to be the same as the initial visit diagnosis, (he could be diagnosed of something that is a complication of the initial diagnosis).

Ideally I would want to add variables that can be overridden to change those assumptions, e.g:
self.readmission_threshold = 30
self.include_any_readmission = True